Example/HorizonCalendarExample/HorizonCalendarExample/Demo View Controllers/MonthFootersDemoViewController.swift

```diff
@@ -0,0 +1,93 @@
+//  Created by Sebastien Audeon on 11/26/20.
+//  Copyright © 2020 Airbnb. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import HorizonCalendar
+
+class MonthFootersDemoViewController: DemoViewController {
+  
+  // MARK: Internal
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    
+    title = "Month Footer Demo"
+    
+    calendarView.daySelectionHandler = { [weak self] day in
+      guard let self = self else { return }
+      
+      self.selectedDay = day
+      self.calendarView.setContent(self.makeContent())
+    }
```

Comment:
```suggestion
```

To keep things really simple, let's remove the selection logic from this file since it just distracts from the month footer stuff for someone trying to figure out how to use that.